server/repository/src/db_diesel/name_link_row.rs

```diff
@@ -13,9 +15,10 @@ table! {
 }
 joinable!(name_link -> name (name_id));
 allow_tables_to_appear_in_same_query!(name_link, item_link);
+allow_tables_to_appear_in_same_query!(name_link, changelog_deduped);
```

Comment:
I would move this to the changelog file since the changelog depends on name_link and this files doesn't need to know about the changelog. Same for item_link...

Reply:
Sweet db14f53787b30c5f0648183466298e2e91b94917

Reply:
Sweet https://github.com/msupply-foundation/open-msupply/commit/db14f53787b30c5f0648183466298e2e91b94917